Src/FluentAssertions/Formatting/DefaultValueFormatter.cs

```diff
@@ -57,48 +61,43 @@ protected virtual MemberInfo[] GetMembers(Type type)
             return type.GetNonPrivateMembers().ToArray();
         }
 
-        /// <summary>
-        /// Selects the name to display for <paramref name="type"/>.
-        /// </summary>
-        /// <param name="type">The <see cref="System.Type"/> of the object being formatted.</param>
-        /// <returns>The name to be displayed for <paramref name="type"/>.</returns>
-        /// <remarks>The default is <see cref="System.Type.FullName"/>.</remarks>
-        protected virtual string TypeDisplayName(Type type) => type.FullName;
-
         private static bool HasDefaultToStringImplementation(object value)
         {
             string str = value.ToString();
 
             return str is null || str == value.GetType().ToString();
         }
 
-        private string GetTypeAndMemberValues(object obj, FormattingContext context, FormatChild formatChild)
+        private void WriteTypeAndMemberValues(object obj, FormattedObjectGraph output, FormatChild formatChild)
```

Comment:
🔧 Consistency if we use `FormattedObjectGraph output` or `FormattedObjectGraph formattedGraph` (multiple places)